backend/package.json

```diff
@@ -19,6 +19,7 @@
     "express": "^4.21.0",
     "express-session": "^1.18.0",
     "nodemailer": "^6.9.15",
+    "nodemon": "^3.1.9",
```

Comment:
Remove nodemon from the package.json file and delete the package-lock.json file from the backend. Then run npm install to update it. Nodemon should be listed in devDependencies, not dependencies.